api/wb/views.py

```diff
@@ -89,4 +100,11 @@ class CopyFileMetadataView(FileMetadataView):
     view_name = 'metadata-copy'
 
     def perform_file_action(self, source, destination, name):
-        return source.copy_under(destination, name)
+        dest_target = destination.target
+        source_target = source.target
+        ret = source.copy_under(destination, name)
+
+        if dest_target != source_target:
+            update_storage_usage(destination.target)
```

Comment:
I don't think we should have this check `if dest_target != source_target`...the storage *should* be updated if I'm copying to the same location - because i have two copies of that same file now, it should go up twice as much!

Please verify this with a test.

Reply:
Also, v1 `osfstorage_copy_hook` does not have this check, for a similar reason.